ElementX/Sources/Services/Client/ClientProxy.swift

```diff
@@ -443,7 +443,7 @@ class ClientProxy: ClientProxyProtocol {
                 .finish()
             let roomListService = syncService.roomListService()
             
-            let roomMessageEventStringBuilder = RoomMessageEventStringBuilder(attributedStringBuilder: AttributedStringBuilder(permalinkBaseURL: .homeDirectory))
+            let roomMessageEventStringBuilder = RoomMessageEventStringBuilder(attributedStringBuilder: AttributedStringBuilder(permalinkBaseURL: .homeDirectory, mentionBuilder: MentionBuilder(mentionsEnabled: appSettings.mentionsEnabled)))
```

Comment:
Might as well fix this whilst we're here…
```suggestion
            let roomMessageEventStringBuilder = RoomMessageEventStringBuilder(attributedStringBuilder: AttributedStringBuilder(permalinkBaseURL: appSettings.permalinkBaseURL, mentionBuilder: MentionBuilder(mentionsEnabled: appSettings.mentionsEnabled)))
```